tests/api/test_security.py

```diff
@@ -0,0 +1,100 @@
+import requests
+import unittest
+import os
+
+
+class TestSecurity(object):
+    def setUp(self):
+        # app['TESTING'] = True
+        # self.app = app.test_client()
+        pass
+
+    def test_main_page(self, flask_api_url):
+        response = requests.get(flask_api_url + '/')
+        assert response.status_code == 200
+
+    def test_login_fake_user(self, flask_api_url):
+        response = requests.post(flask_api_url + '/api/users/login',
+                                 data=dict(email='user1', password='password'))
+        print(response)
+        assert response.reason == 'INTERNAL SERVER ERROR'
```

Comment:
This is very poor behaviour (not the test methods fault, but this has to be changed too), a login attempt with wrong user/password should not lead to an internal server error but a 400 code